src/Uno.UI/Microsoft/UI/Xaml/Controls/TreeView/TreeViewItemAutomationPeer.cs

```diff
@@ -0,0 +1,293 @@
+﻿using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+using Uno.UI.Helpers.WinUI;
+using Windows.UI.Xaml.Automation.Provider;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.UI.Xaml.Automation.Peers
+{
+	public partial class TreeViewItemAutomationPeer : ListViewItemAutomationPeer, IExpandCollapseProvider
+	{
+		private readonly TreeViewItem _owner;
+
+		public TreeViewItemAutomationPeer(TreeViewItem owner) : base(owner)
+		{
+		}
+
+		// IExpandCollapseProvider 
+		public ExpandCollapseState ExpandCollapseState
+		{
+			get
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)